packages/stark-ui/src/modules/table/components/table.component.ts

```diff
@@ -251,30 +251,30 @@ export class StarkTableComponent extends AbstractStarkUiComponent implements OnI
 	 * Reference to the MatTable embedded in this component
 	 */
 	@ViewChild(MatTable)
-	public table: MatTable<object>;
+	public table!: MatTable<object>;
 
 	/**
 	 * Reference to the MatPaginator embedded in this component
 	 */
 	@ViewChild(StarkPaginationComponent)
-	public starkPaginator: StarkPaginationComponent;
+	public starkPaginator!: StarkPaginationComponent;
 
 	/**
 	 * Columns added automatically by this component according to the columnProperties input
 	 */
 	@ViewChildren(StarkTableColumnComponent)
-	public viewColumns: QueryList<StarkTableColumnComponent>;
+	public viewColumns!: QueryList<StarkTableColumnComponent>;
 
 	/**
 	 * Columns added by the user via transclusion inside an <div> element with class "stark-table-columns"
 	 */
 	@ContentChildren(StarkTableColumnComponent)
-	public contentColumns: QueryList<StarkTableColumnComponent>;
+	public contentColumns: QueryList<StarkTableColumnComponent> = new QueryList();
```

Comment:
I think we should simply use the `!` modifier here to avoid creating a dummy `QueryList` which might consume some memory which at the end will not be used